apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue

```diff
@@ -277,11 +279,46 @@
             >
           </p>
         </ContentBlock>
+
+        <ContentBlock id="version" title="Version">
+          <DefinitionList>
+            <template v-for="[term, definition] in Object.entries(manifest)">
+              <DefinitionListTerm>{{ term }}</DefinitionListTerm>
+              <DefinitionListDefinition>
+                {{ definition }}
+              </DefinitionListDefinition>
+            </template>
+          </DefinitionList>
+        </ContentBlock>
       </ContentBlocks>
     </template>
   </LayoutsDetailPage>
 </template>
 
-<script setup>
+<script setup lang="ts">
 useHead({ title: "About" });
+const { data, error } = await useFetch(
+  `/${useRoute().params.schema}/api/graphql`,
+  {
+    key: `manifest`,
+    baseURL: useRuntimeConfig().public.apiBase,
+    method: "POST",
+    body: {
+      query: ` 
+    {
+      _manifest {
+        ImplementationVersion
+        SpecificationVersion
+        DatabaseVersion
```

Comment:
This is nice

I supposed you also would like the catalogue version, i.e. the _schema.tables(name=Version).description?

Reply:
yes , thats is what we needed , but then i thought lets add what we have for now

Reply:
okay, I approve this one first and this other thing can be next PR